### PR TITLE
Add common lambda handler code to library

### PR DESCRIPTION
**Pull request by andreyshuster**
Add AWS Lambda daemon handler to library
See context here  https://github.com/globality-corp/uhura/pull/64
@tonybajan-globality @afallou  @rh-glob 
Please suggest how it could be tested better.

### CODE REVIEW

File: setup.py [hunk 1]
```diff
@@ -26,6 +26,7 @@
         "microcosm-caching>=0.2.0",
         "microcosm-daemon>=1.0.0",
         "microcosm-logging>=1.3.0",
+        "isort<5.0.0",
```
Comment (rh-glob) [inline setup.py:29]: Is this required for build to pass? Ideally we wouldn't have such a restriction in a library, as it makes it more complicated to to upgrade isort in the future for the services using this library.

Comment (andreyshuster) [reply setup.py:29]: Unfortunately. 
`Lint` job fails because of that.
We have this issue in all repositories at this moment.

Comment (andreyshuster) [reply setup.py:29]: Just saw Tony's comment in Slack on that. let me try to kick the build again and see what happens

Comment (andreyshuster) [reply setup.py]: Looks like it's not required anymore!

**Pull request merged**